src/main/java/org/team1540/bobafett/commands/drivetrain/DriveToAprilTag.java

```diff
@@ -28,9 +34,17 @@ public DriveToAprilTag(Drivetrain drivetrain, ChickenPhotonCamera camera, int ta
     @Override
     public void initialize() {
         aprilTag = camera.getTarget(targetID);
-        if (aprilTag != null) drivetrain.setPercent(0.5, 0.5);
-        originalBrakeMode = drivetrain.getBrakeType();
+        originalBrakeMode = drivetrain.getBrakeMode();
         drivetrain.setBrakeMode(NeutralMode.Brake);
+        drivetrain.setYaw(0);
+    }
+
+    @Override
+    public void execute() {
+        if (aprilTag != null) {
+            double out = pidController.calculate(drivetrain.getYaw());
+            drivetrain.setPercent(0.5 - out, 0.5 + out);
```

Comment:
I don't remember if you finished testing this, does it work?

Reply:
it _should_ work since it uses the same concept as the turning to AprilTag, just this time I'm adding pid output to the motor speed instead of setting the motor speed to the pid output. The error should be relatively small so there shouldn't be any problems, but I might have to tune another pid...